app/views/images/index.html.erb

```diff
@@ -1,9 +1,14 @@
-<div class="container-fluid album">
+<section class="container-fluid">
+  <% ActsAsTaggableOn::Tag.all.map(&:name).each do |tag| %>
```

Comment:
First I think you could do `ActsAsTaggableOn::Tag.all.map(&:name).join(' ')`. And second I think this is to display all tags that exists right? I think this story is to display each image's tags below each image on the index page. So I think you should add this part below line 10 and use `@image.tag_list`.